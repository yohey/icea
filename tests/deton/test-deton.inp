reac oxid O2 wt%=100 t(k)=298.15
     fuel H2 wt%=100 t(k)=298.15

prob detonation case=Test-DETON t=300,350,400,450,500, r,e=1, pbar=1,5,10,15,20

output calories transport
end
