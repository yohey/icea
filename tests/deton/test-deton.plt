#  sonic_1     gamma_1     h_1         t_1         p_1         detvel      mach        
   5.3949E+02  1.4014E+00  1.0685E+00  3.0000E+02  9.8692E-01  2.8349E+03  5.2548E+00
   5.3949E+02  1.4014E+00  1.0685E+00  3.0000E+02  4.9346E+00  2.9218E+03  5.4158E+00
   5.3949E+02  1.4014E+00  1.0685E+00  3.0000E+02  9.8692E+00  2.9581E+03  5.4832E+00
   5.3949E+02  1.4014E+00  1.0685E+00  3.0000E+02  1.4804E+01  2.9789E+03  5.5216E+00
   5.3949E+02  1.4014E+00  1.0685E+00  3.0000E+02  1.9738E+01  2.9933E+03  5.5484E+00
   5.8167E+02  1.3964E+00  3.0090E+01  3.5000E+02  9.8692E-01  2.8196E+03  4.8475E+00
   5.8167E+02  1.3964E+00  3.0090E+01  3.5000E+02  4.9346E+00  2.9076E+03  4.9987E+00
   5.8167E+02  1.3964E+00  3.0090E+01  3.5000E+02  9.8692E+00  2.9445E+03  5.0622E+00
   5.8167E+02  1.3964E+00  3.0090E+01  3.5000E+02  1.4804E+01  2.9657E+03  5.0985E+00
   5.8167E+02  1.3964E+00  3.0090E+01  3.5000E+02  1.9738E+01  2.9804E+03  5.1239E+00
   6.2097E+02  1.3925E+00  5.9340E+01  4.0000E+02  9.8692E-01  2.8052E+03  4.5175E+00
   6.2097E+02  1.3925E+00  5.9340E+01  4.0000E+02  4.9346E+00  2.8943E+03  4.6609E+00
   6.2097E+02  1.3925E+00  5.9340E+01  4.0000E+02  9.8692E+00  2.9318E+03  4.7213E+00
   6.2097E+02  1.3925E+00  5.9340E+01  4.0000E+02  1.4804E+01  2.9534E+03  4.7560E+00
   6.2097E+02  1.3925E+00  5.9340E+01  4.0000E+02  1.9738E+01  2.9684E+03  4.7803E+00
   6.5783E+02  1.3891E+00  8.8784E+01  4.5000E+02  9.8692E-01  2.7915E+03  4.2435E+00
   6.5783E+02  1.3891E+00  8.8784E+01  4.5000E+02  4.9346E+00  2.8817E+03  4.3806E+00
   6.5783E+02  1.3891E+00  8.8784E+01  4.5000E+02  9.8692E+00  2.9197E+03  4.4384E+00
   6.5783E+02  1.3891E+00  8.8784E+01  4.5000E+02  1.4804E+01  2.9416E+03  4.4716E+00
   6.5783E+02  1.3891E+00  8.8784E+01  4.5000E+02  1.9738E+01  2.9568E+03  4.4948E+00
   6.9259E+02  1.3858E+00  1.1841E+02  5.0000E+02  9.8692E-01  2.7783E+03  4.0115E+00
   6.9259E+02  1.3858E+00  1.1841E+02  5.0000E+02  4.9346E+00  2.8695E+03  4.1432E+00
   6.9259E+02  1.3858E+00  1.1841E+02  5.0000E+02  9.8692E+00  2.9081E+03  4.1989E+00
   6.9259E+02  1.3858E+00  1.1841E+02  5.0000E+02  1.4804E+01  2.9303E+03  4.2309E+00
   6.9259E+02  1.3858E+00  1.1841E+02  5.0000E+02  1.9738E+01  2.9458E+03  4.2533E+00
#  sonic_1     gamma_1     h_1         t_1         p_1         detvel      mach        
