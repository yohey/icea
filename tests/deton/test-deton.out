
 *******************************************************************************

         NASA-GLENN CHEMICAL EQUILIBRIUM PROGRAM CEA2, MAY 21, 2004
                   BY  BONNIE MCBRIDE AND SANFORD GORDON
      REFS: NASA RP-1311, PART I, 1994 AND NASA RP-1311, PART II, 1996

 *******************************************************************************



 reac oxid O2 wt%=100 t(k)=298.15
      fuel H2 wt%=100 t(k)=298.15
  
 prob detonation case=Test-DETON t=300,350,400,450,500, r,e=1, pbar=1,5,10,15,20
  
 output calories transport
 end

 OPTIONS: TP=F  HP=F  SP=F  TV=F  UV=F  SV=F  DETN=T  SHOCK=F  REFL=F  INCD=F
 RKT=F  FROZ=F  EQL=F  IONS=F  SIUNIT=F  DEBUGF=F  SHKDBG=F  DETDBG=F  TRNSPT=T

 T,K =   300.0000   350.0000   400.0000   450.0000   500.0000

 TRACE= 0.00E+00  S/R= 0.000000E+00  H/R= 0.000000E+00  U/R= 0.000000E+00

 P,BAR =     1.000000     5.000000    10.000000    15.000000    20.000000

    REACTANT          WT.FRAC   (ENERGY/R),K   TEMP,K  DENSITY
        EXPLODED FORMULA
 O: O2               1.000000  -0.154035E-05   298.15  0.0000
          O  2.00000
 F: H2               1.000000  -0.326752E-06   298.15  0.0000
          H  2.00000

  SPECIES BEING CONSIDERED IN THIS SYSTEM
 (CONDENSED PHASE MAY HAVE NAME LISTED SEVERAL TIMES)
  LAST thermo.inp UPDATE:    9/09/04

  g 6/97  *H               g 4/02  HO2              tpis78  *H2            
  g 8/89  H2O              g 6/99  H2O2             g 5/97  *O             
  g 4/02  *OH              tpis89  *O2              g 8/01  O3             
  g11/99  H2O(cr)          g 8/01  H2O(L)           g 8/01  H2O(L)         

 SPECIES WITH TRANSPORT PROPERTIES

        PURE SPECIES

  H                 H2                H2O               O               
  OH                O2              

     BINARY INTERACTIONS

     H               H2              
     H               O               
     H2              H2O             
     H2              O2              
     H2O             O2              
     O               O2              


 O/F =   7.936683

                       EFFECTIVE FUEL     EFFECTIVE OXIDANT        MIXTURE
 ENTHALPY                  h(2)/R              h(1)/R               h0/R
 (KG-MOL)(K)/KG       -0.16208918E-06     -0.48137850E-07     -0.60888815E-07

 KG-FORM.WT./KG             bi(2)               bi(1)               b0i
  *O                   0.00000000E+00      0.62502344E-01      0.55508435E-01
  *H                   0.99212255E+00      0.00000000E+00      0.11101687E+00

 POINT ITN      T            O           H 
   1    8    3605.449     -15.691     -10.322

 POINT ITN      T            O           H 
   1    3    3632.365     -15.616     -10.242

 POINT ITN      T            O           H 
   1    3    3674.556     -15.617     -10.238

 POINT ITN      T            O           H 
   1    2    3673.327     -15.617     -10.238
   2    5    3887.952     -15.101      -9.689
   2    3    3934.469     -15.011      -9.592
   2    3    3992.511     -15.014      -9.587
   2    2    3990.766     -15.014      -9.587
   3    5    4014.548     -14.851      -9.421
   3    3    4072.124     -14.756      -9.318
   3    3    4136.900     -14.759      -9.313
   3    2    4134.985     -14.759      -9.313
   4    5    4089.482     -14.706      -9.266
   4    3    4154.363     -14.608      -9.159
   4    3    4222.900     -14.611      -9.154
   4    2    4220.911     -14.611      -9.154
   5    5    4142.896     -14.604      -9.156
   5    3    4213.351     -14.504      -9.047
   5    3    4284.435     -14.507      -9.042
   5    2    4282.408     -14.507      -9.042
   6    5    3636.301     -15.698     -10.325
   6    3    3639.454     -15.677     -10.303
   6    3    3650.229     -15.678     -10.302
   6    2    3650.105     -15.678     -10.302
   7    5    3927.452     -15.109      -9.692
   7    3    3943.243     -15.071      -9.651
   7    3    3967.132     -15.072      -9.650
   7    2    3966.753     -15.073      -9.650
   8    5    4058.524     -14.860      -9.424
   8    3    4081.751     -14.815      -9.376
   8    3    4111.562     -14.817      -9.374
   8    2    4111.060     -14.817      -9.374


                     DETONATION PROPERTIES OF AN IDEAL REACTING GAS
 CASE = Test-DETON     

             REACTANT                    WT FRACTION      ENERGY      TEMP
                                          (SEE NOTE)      CAL/MOL       K  
 OXIDANT     O2                           1.0000000        -0.000    298.150
 FUEL        H2                           1.0000000        -0.000    298.150

 O/F=    7.93668  %FUEL= 11.189834  R,EQ.RATIO= 1.000000  PHI,EQ.RATIO= 1.000000

 UNBURNED GAS

 P1, ATM           0.9869   4.9346   9.8692  14.8038  19.7385   0.9869   4.9346   9.8692
 T1, K             300.00   300.00   300.00   300.00   300.00   350.00   350.00   350.00
 H1, CAL/G           1.07     1.07     1.07     1.07     1.07    30.09    30.09    30.09
 M1, (1/n)         12.010   12.010   12.010   12.010   12.010   12.010   12.010   12.010
 GAMMA1            1.4014   1.4014   1.4014   1.4014   1.4014   1.3964   1.3964   1.3964
 SON VEL1,M/SEC     539.5    539.5    539.5    539.5    539.5    581.7    581.7    581.7

 BURNED GAS

 P, ATM            18.404   97.217   198.87   302.14   406.43   15.681   82.893   169.63
 T, K             3673.33  3990.77  4134.99  4220.91  4282.41  3650.10  3966.75  4111.06
 RHO, G/CC       8.8518-4 4.4169-3 8.8266-3 1.3234-2 1.7640-2 7.5553-4 3.7710-3 7.5365-3
 H, CAL/G          677.35   718.20   735.64   745.69   752.74   696.68   737.87   755.51
 U, CAL/G          173.83   185.18   190.01   192.81   194.76   194.07   205.53   210.42
 G, CAL/G        -14625.1 -15050.2 -15227.5 -15327.2 -15395.5 -14608.6 -15039.9 -15221.5
 S, CAL/(G)(K)     4.1658   3.9512   3.8605   3.8079   3.7708   4.1931   3.9775   3.8864

 M, (1/n)          14.497   14.878   15.060   15.171   15.252   14.432   14.808   14.988
 (dLV/dLP)t      -1.08202 -1.07073 -1.06566 -1.06265 -1.06050 -1.08387 -1.07261 -1.06754
 (dLV/dLT)p        2.3596   2.0881   1.9785   1.9167   1.8739   2.3977   2.1225   2.0112
 Cp, CAL/(G)(K)    3.8920   3.0428   2.7338   2.5681   2.4572   4.0100   3.1357   2.8167
 GAMMAs            1.1288   1.1372   1.1406   1.1425   1.1438   1.1281   1.1366   1.1401
 SON VEL,M/SEC     1542.1   1592.5   1613.7   1625.7   1634.1   1540.2   1591.1   1612.5

 TRANSPORT PROPERTIES (GASES ONLY)
   CONDUCTIVITY IN UNITS OF MILLICALORIES/(CM)(K)(SEC)

 VISC,MILLIPOISE   1.1409   1.2115   1.2426   1.2609   1.2738   1.1357   1.2063   1.2376

  WITH EQUILIBRIUM REACTIONS

 Cp, CAL/(G)(K)    3.8920   3.0428   2.7338   2.5681   2.4572   4.0100   3.1357   2.8167
 CONDUCTIVITY      9.1133   7.3068   6.6091   6.2248   5.9630   9.3771   7.5330   6.8184
 PRANDTL NUMBER    0.4873   0.5045   0.5140   0.5202   0.5249   0.4857   0.5022   0.5112

  WITH FROZEN REACTIONS

 Cp, CAL/(G)(K)    0.7784   0.7866   0.7901   0.7921   0.7936   0.7778   0.7860   0.7895
 CONDUCTIVITY      1.2902   1.3611   1.3913   1.4088   1.4210   1.2871   1.3585   1.3890
 PRANDTL NUMBER    0.6883   0.7001   0.7056   0.7090   0.7114   0.6863   0.6980   0.7035

 DETONATION PARAMETERS

 P/P1              18.648   19.701   20.150   20.409   20.591   15.888   16.798   17.188
 T/T1              12.244   13.303   13.783   14.070   14.275   10.429   11.334   11.746
 M/M1              1.2071   1.2388   1.2539   1.2632   1.2699   1.2016   1.2329   1.2479
 RHO/RHO1          1.8384   1.8347   1.8332   1.8324   1.8318   1.8307   1.8274   1.8261
 DET MACH NUMBER   5.2548   5.4158   5.4832   5.5216   5.5484   4.8475   4.9987   5.0622
 DET VEL,M/SEC     2834.9   2921.8   2958.1   2978.9   2993.3   2819.6   2907.6   2944.5

 MOLE FRACTIONS

 *H               0.08019  0.06216  0.05459  0.05027  0.04726  0.08313  0.06491  0.05723
 HO2              0.00018  0.00038  0.00051  0.00060  0.00068  0.00017  0.00036  0.00048
 *H2              0.16216  0.15426  0.14958  0.14646  0.14406  0.16349  0.15609  0.15165
 H2O              0.53163  0.56938  0.58857  0.60064  0.60959  0.52491  0.56168  0.58044
 H2O2             0.00002  0.00006  0.00010  0.00013  0.00016  0.00002  0.00006  0.00009
 *O               0.03748  0.03012  0.02682  0.02488  0.02352  0.03875  0.03138  0.02806
 *OH              0.14145  0.14179  0.14031  0.13890  0.13766  0.14202  0.14298  0.14180
 *O2              0.04688  0.04185  0.03952  0.03811  0.03708  0.04751  0.04255  0.04025

  * THERMODYNAMIC PROPERTIES FITTED TO 20000.K

    PRODUCTS WHICH WERE CONSIDERED BUT WHOSE MOLE FRACTIONS
    WERE LESS THAN 5.000000E-06 FOR ALL ASSIGNED CONDITIONS

 O3              H2O(cr)         H2O(L)         

 NOTE. WEIGHT FRACTION OF FUEL IN TOTAL FUELS AND OF OXIDANT IN TOTAL OXIDANTS





 POINT ITN      T            O           H 
   1    5    4136.300     -14.716      -9.269

 POINT ITN      T            O           H 
   1    3    4164.525     -14.667      -9.217

 POINT ITN      T            O           H 
   1    3    4197.770     -14.669      -9.215

 POINT ITN      T            O           H 
   1    2    4197.200     -14.669      -9.215
   2    5    4191.831     -14.613      -9.160
   2    3    4223.911     -14.563      -9.105
   2    3    4259.546     -14.565      -9.103
   2    2    4258.931     -14.565      -9.103
   3    5    3666.156     -15.706     -10.329
   3    3    3647.196     -15.730     -10.356
   3    3    3630.794     -15.731     -10.358
   3    2    3630.728     -15.730     -10.358
   4    5    3965.733     -15.118      -9.696
   4    3    3952.866     -15.123      -9.703
   4    3    3946.793     -15.124      -9.704
   4    1    3946.813     -15.124      -9.704
   5    5    4101.181     -14.869      -9.428
   5    3    4092.323     -14.867      -9.427
   5    2    4091.243     -14.868      -9.428
   6    5    4181.742     -14.725      -9.273
   6    3    4175.693     -14.719      -9.267
   6    2    4177.619     -14.720      -9.268
   7    5    4239.352     -14.623      -9.164
   7    3    4235.519     -14.614      -9.155
   7    2    4239.594     -14.615      -9.156
   8    5    3695.083     -15.714     -10.333
   8    3    3655.464     -15.778     -10.403
   8    3    3614.980     -15.778     -10.408
   8    2    3614.248     -15.777     -10.408


                     DETONATION PROPERTIES OF AN IDEAL REACTING GAS
 CASE = Test-DETON     

             REACTANT                    WT FRACTION      ENERGY      TEMP
                                          (SEE NOTE)      CAL/MOL       K  
 OXIDANT     O2                           1.0000000        -0.000    298.150
 FUEL        H2                           1.0000000        -0.000    298.150

 O/F=    7.93668  %FUEL= 11.189834  R,EQ.RATIO= 1.000000  PHI,EQ.RATIO= 1.000000

 UNBURNED GAS

 P1, ATM          14.8038  19.7385   0.9869   4.9346   9.8692  14.8038  19.7385   0.9869
 T1, K             350.00   350.00   400.00   400.00   400.00   400.00   400.00   450.00
 H1, CAL/G          30.09    30.09    59.34    59.34    59.34    59.34    59.34    88.78
 M1, (1/n)         12.010   12.010   12.010   12.010   12.010   12.010   12.010   12.010
 GAMMA1            1.3964   1.3964   1.3925   1.3925   1.3925   1.3925   1.3925   1.3891
 SON VEL1,M/SEC     581.7    581.7    621.0    621.0    621.0    621.0    621.0    657.8

 BURNED GAS

 P, ATM            257.78   346.83   13.647   72.194   147.79   224.64   302.30   12.070
 T, K             4197.20  4258.93  3630.73  3946.81  4091.24  4177.62  4239.59  3614.25
 RHO, G/CC       1.1300-2 1.5063-2 6.5824-4 3.2863-3 6.5683-3 9.8493-3 1.3130-2 5.8252-4
 H, CAL/G          765.70   772.86   716.71   758.21   776.03   786.39   793.68   737.26
 U, CAL/G          213.26   215.25   214.64   226.20   231.14   234.04   236.09   235.46
 G, CAL/G        -15324.3 -15395.2 -14596.9 -15033.9 -15219.4 -15325.1 -15398.1 -14588.8
 S, CAL/(G)(K)     3.8335   3.7963   4.2178   4.0012   3.9097   3.8566   3.8192   4.2405

 M, (1/n)          15.098   15.178   14.371   14.742   14.921   15.030   15.110   14.313
 (dLV/dLP)t      -1.06453 -1.06237 -1.08560 -1.07436 -1.06928 -1.06627 -1.06411 -1.08722
 (dLV/dLT)p        1.9484   1.9048   2.4327   2.1541   2.0412   1.9774   1.9332   2.4652
 Cp, CAL/(G)(K)    2.6455   2.5307   4.1187   3.2214   2.8932   2.7168   2.5985   4.2203
 GAMMAs            1.1420   1.1433   1.1275   1.1361   1.1396   1.1416   1.1429   1.1270
 SON VEL,M/SEC     1624.7   1633.2   1539.0   1590.3   1611.9   1624.3   1632.9   1538.3

 TRANSPORT PROPERTIES (GASES ONLY)
   CONDUCTIVITY IN UNITS OF MILLICALORIES/(CM)(K)(SEC)

 VISC,MILLIPOISE   1.2559   1.2690   1.1314   1.2021   1.2334   1.2519   1.2650   1.1276

  WITH EQUILIBRIUM REACTIONS

 Cp, CAL/(G)(K)    2.6455   2.5307   4.1187   3.2214   2.8932   2.7168   2.5985   4.2203
 CONDUCTIVITY      6.4241   6.1551   9.6206   7.7423   7.0123   6.6088   6.3333   9.8485
 PRANDTL NUMBER    0.5172   0.5217   0.4844   0.5002   0.5089   0.5146   0.5190   0.4832

  WITH FROZEN REACTIONS

 Cp, CAL/(G)(K)    0.7916   0.7931   0.7773   0.7855   0.7891   0.7912   0.7926   0.7769
 CONDUCTIVITY      1.4066   1.4190   1.2849   1.3567   1.3874   1.4052   1.4177   1.2833
 PRANDTL NUMBER    0.7068   0.7092   0.6844   0.6960   0.7015   0.7048   0.7072   0.6826

 DETONATION PARAMETERS

 P/P1              17.413   17.571   13.827   14.630   14.975   15.175   15.315   12.230
 T/T1              11.992   12.168    9.077    9.867   10.228   10.444   10.599    8.032
 M/M1              1.2571   1.2638   1.1965   1.2275   1.2423   1.2514   1.2581   1.1917
 RHO/RHO1          1.8254   1.8249   1.8228   1.8200   1.8189   1.8183   1.8179   1.8147
 DET MACH NUMBER   5.0985   5.1239   4.5175   4.6609   4.7213   4.7560   4.7803   4.2435
 DET VEL,M/SEC     2965.7   2980.4   2805.2   2894.3   2931.8   2953.4   2968.4   2791.5

 MOLE FRACTIONS

 *H               0.05282  0.04975  0.08588  0.06749  0.05969  0.05522  0.05209  0.08848
 HO2              0.00057  0.00064  0.00016  0.00034  0.00046  0.00054  0.00061  0.00015
 *H2              0.14867  0.14637  0.16469  0.15774  0.15352  0.15066  0.14846  0.16578
 H2O              0.59227  0.60104  0.51868  0.55459  0.57297  0.58457  0.59319  0.51283
 H2O2             0.00012  0.00015  0.00002  0.00005  0.00008  0.00011  0.00013  0.00001
 *O               0.02610  0.02471  0.03994  0.03256  0.02921  0.02724  0.02583  0.04106
 *OH              0.14060  0.13949  0.14256  0.14407  0.14317  0.14214  0.14116  0.14309
 *O2              0.03885  0.03784  0.04807  0.04317  0.04090  0.03952  0.03852  0.04858

  * THERMODYNAMIC PROPERTIES FITTED TO 20000.K

    PRODUCTS WHICH WERE CONSIDERED BUT WHOSE MOLE FRACTIONS
    WERE LESS THAN 5.000000E-06 FOR ALL ASSIGNED CONDITIONS

 O3              H2O(cr)         H2O(L)         

 NOTE. WEIGHT FRACTION OF FUEL IN TOTAL FUELS AND OF OXIDANT IN TOTAL OXIDANTS





 POINT ITN      T            O           H 
   1    5    4002.876     -15.127      -9.700

 POINT ITN      T            O           H 
   1    3    3963.182     -15.170      -9.748

 POINT ITN      T            O           H 
   1    3    3930.219     -15.170      -9.752

 POINT ITN      T            O           H 
   1    2    3929.946     -15.170      -9.752
   2    5    4142.607     -14.879      -9.433
   2    3    4103.672     -14.913      -9.472
   2    3    4074.689     -14.913      -9.476
   2    2    4074.549     -14.913      -9.476
   3    5    4225.899     -14.735      -9.278
   3    3    4187.688     -14.765      -9.312
   3    3    4161.210     -14.765      -9.315
   3    2    4161.129     -14.764      -9.315
   4    5    4285.549     -14.634      -9.169
   4    3    4247.991     -14.660      -9.200
   4    3    4223.357     -14.660      -9.202
   4    2    4223.310     -14.660      -9.202
   5    5    3723.152     -15.723     -10.338
   5    3    3664.160     -15.821     -10.445
   5    3    3601.949     -15.820     -10.453
   5    2    3600.027     -15.819     -10.452
   6    5    4038.965     -15.137      -9.706
   6    3    3974.068     -15.212      -9.789
   6    3    3916.561     -15.211      -9.796
   6    2    3915.480     -15.211      -9.795
   7    5    4182.890     -14.889      -9.438
   7    3    4115.660     -14.955      -9.513
   7    3    4061.061     -14.954      -9.518
   7    2    4060.279     -14.954      -9.518
   8    5    4268.861     -14.746      -9.283
   8    3    4200.367     -14.806      -9.353
   8    3    4147.717     -14.805      -9.358
   8    2    4147.087     -14.805      -9.358


                     DETONATION PROPERTIES OF AN IDEAL REACTING GAS
 CASE = Test-DETON     

             REACTANT                    WT FRACTION      ENERGY      TEMP
                                          (SEE NOTE)      CAL/MOL       K  
 OXIDANT     O2                           1.0000000        -0.000    298.150
 FUEL        H2                           1.0000000        -0.000    298.150

 O/F=    7.93668  %FUEL= 11.189834  R,EQ.RATIO= 1.000000  PHI,EQ.RATIO= 1.000000

 UNBURNED GAS

 P1, ATM           4.9346   9.8692  14.8038  19.7385   0.9869   4.9346   9.8692  14.8038
 T1, K             450.00   450.00   450.00   450.00   500.00   500.00   500.00   500.00
 H1, CAL/G          88.78    88.78    88.78    88.78   118.41   118.41   118.41   118.41
 M1, (1/n)         12.010   12.010   12.010   12.010   12.010   12.010   12.010   12.010
 GAMMA1            1.3891   1.3891   1.3891   1.3891   1.3858   1.3858   1.3858   1.3858
 SON VEL1,M/SEC     657.8    657.8    657.8    657.8    692.6    692.6    692.6    692.6

 BURNED GAS

 P, ATM            63.900   130.86   198.94   267.75   10.813   57.285   117.35   178.44
 T, K             3929.95  4074.55  4161.13  4223.31  3600.03  3915.48  4060.28  4147.09
 RHO, G/CC       2.9091-3 5.8151-3 8.7201-3 1.1624-2 5.2191-4 2.6072-3 5.2122-3 7.8165-3
 H, CAL/G          779.08   797.10   807.55   814.90   758.26   800.38   818.59   829.16
 U, CAL/G          247.13   252.14   255.05   257.10   256.51   268.28   273.35   276.29
 G, CAL/G        -15031.2 -15220.3 -15328.4 -15403.5 -14583.5 -15031.0 -15223.4 -15333.9
 S, CAL/(G)(K)     4.0230   3.9311   3.8778   3.8402   4.2616   4.0433   3.9510   3.8975

 M, (1/n)          14.681   14.858   14.967   15.046   14.258   14.623   14.798   14.906
 (dLV/dLP)t      -1.07600 -1.07092 -1.06790 -1.06573 -1.08877 -1.07756 -1.07248 -1.06945
 (dLV/dLT)p        2.1835   2.0692   2.0045   1.9596   2.4959   2.2112   2.0955   2.0299
 Cp, CAL/(G)(K)    3.3015   2.9647   2.7836   2.6619   4.3162   3.3772   3.0324   2.8467
 GAMMAs            1.1357   1.1392   1.1412   1.1426   1.1266   1.1353   1.1389   1.1409
 SON VEL,M/SEC     1589.9   1611.7   1624.2   1633.0   1537.9   1589.8   1611.9   1624.6

 TRANSPORT PROPERTIES (GASES ONLY)
   CONDUCTIVITY IN UNITS OF MILLICALORIES/(CM)(K)(SEC)

 VISC,MILLIPOISE   1.1985   1.2299   1.2485   1.2617   1.1244   1.1954   1.2269   1.2456

  WITH EQUILIBRIUM REACTIONS

 Cp, CAL/(G)(K)    3.3015   2.9647   2.7836   2.6619   4.3162   3.3772   3.0324   2.8467
 CONDUCTIVITY      7.9384   7.1942   6.7823   6.5007  10.0641   8.1243   7.3668   6.9469
 PRANDTL NUMBER    0.4984   0.5068   0.5124   0.5166   0.4822   0.4969   0.5050   0.5104

  WITH FROZEN REACTIONS

 Cp, CAL/(G)(K)    0.7851   0.7887   0.7908   0.7923   0.7765   0.7848   0.7884   0.7905
 CONDUCTIVITY      1.3555   1.3865   1.4045   1.4171   1.2823   1.3549   1.3861   1.4043
 PRANDTL NUMBER    0.6942   0.6996   0.7029   0.7054   0.6809   0.6924   0.6978   0.7011

 DETONATION PARAMETERS

 P/P1              12.949   13.259   13.439   13.565   10.957   11.609   11.890   12.054
 T/T1               8.733    9.055    9.247    9.385    7.200    7.831    8.121    8.294
 M/M1              1.2224   1.2371   1.2462   1.2528   1.1872   1.2175   1.2322   1.2411
 RHO/RHO1          1.8125   1.8116   1.8110   1.8107   1.8066   1.8049   1.8042   1.8038
 DET MACH NUMBER   4.3806   4.4384   4.4716   4.4948   4.0115   4.1432   4.1989   4.2309
 DET VEL,M/SEC     2881.7   2919.7   2941.6   2956.8   2778.3   2869.5   2908.1   2930.3

 MOLE FRACTIONS

 *H               0.06992  0.06203  0.05749  0.05431  0.09096  0.07226  0.06428  0.05967
 HO2              0.00032  0.00044  0.00052  0.00059  0.00015  0.00031  0.00042  0.00050
 *H2              0.15925  0.15522  0.15249  0.15037  0.16681  0.16065  0.15680  0.15418
 H2O              0.54796  0.56599  0.57740  0.58588  0.50727  0.54169  0.55940  0.57062
 H2O2             0.00005  0.00008  0.00010  0.00012  0.00001  0.00004  0.00007  0.00010
 *O               0.03368  0.03031  0.02832  0.02690  0.04214  0.03475  0.03137  0.02936
 *OH              0.14508  0.14444  0.14356  0.14270  0.14360  0.14604  0.14563  0.14490
 *O2              0.04373  0.04148  0.04012  0.03913  0.04905  0.04425  0.04202  0.04068

  * THERMODYNAMIC PROPERTIES FITTED TO 20000.K

    PRODUCTS WHICH WERE CONSIDERED BUT WHOSE MOLE FRACTIONS
    WERE LESS THAN 5.000000E-06 FOR ALL ASSIGNED CONDITIONS

 O3              H2O(cr)         H2O(L)         

 NOTE. WEIGHT FRACTION OF FUEL IN TOTAL FUELS AND OF OXIDANT IN TOTAL OXIDANTS





 POINT ITN      T            O           H 
   1    5    4330.516     -14.644      -9.174

 POINT ITN      T            O           H 
   1    3    4261.181     -14.701      -9.240

 POINT ITN      T            O           H 
   1    3    4210.020     -14.700      -9.245

 POINT ITN      T            O           H 
   1    2    4209.489     -14.700      -9.245


                     DETONATION PROPERTIES OF AN IDEAL REACTING GAS
 CASE = Test-DETON     

             REACTANT                    WT FRACTION      ENERGY      TEMP
                                          (SEE NOTE)      CAL/MOL       K  
 OXIDANT     O2                           1.0000000        -0.000    298.150
 FUEL        H2                           1.0000000        -0.000    298.150

 O/F=    7.93668  %FUEL= 11.189834  R,EQ.RATIO= 1.000000  PHI,EQ.RATIO= 1.000000

 UNBURNED GAS

 P1, ATM          19.7385
 T1, K             500.00
 H1, CAL/G         118.41
 M1, (1/n)         12.010
 GAMMA1            1.3858
 SON VEL1,M/SEC     692.6

 BURNED GAS

 P, ATM            240.20
 T, K             4209.49
 RHO, G/CC       1.0420-2
 H, CAL/G          836.60
 U, CAL/G          278.37
 G, CAL/G        -15410.9
 S, CAL/(G)(K)     3.8597

 M, (1/n)          14.985
 (dLV/dLP)t      -1.06728
 (dLV/dLT)p        1.9844
 Cp, CAL/(G)(K)    2.7219
 GAMMAs            1.1423
 SON VEL,M/SEC     1633.4

 TRANSPORT PROPERTIES (GASES ONLY)
   CONDUCTIVITY IN UNITS OF MILLICALORIES/(CM)(K)(SEC)

 VISC,MILLIPOISE   1.2589

  WITH EQUILIBRIUM REACTIONS

 Cp, CAL/(G)(K)    2.7219
 CONDUCTIVITY      6.6597
 PRANDTL NUMBER    0.5145

  WITH FROZEN REACTIONS

 Cp, CAL/(G)(K)    0.7920
 CONDUCTIVITY      1.4170
 PRANDTL NUMBER    0.7036

 DETONATION PARAMETERS

 P/P1              12.169
 T/T1               8.419
 M/M1              1.2477
 RHO/RHO1          1.8035
 DET MACH NUMBER   4.2533
 DET VEL,M/SEC     2945.8

 MOLE FRACTIONS

 *H               0.05644
 HO2              0.00057
 *H2              0.15214
 H2O              0.57898
 H2O2             0.00012
 *O               0.02792
 *OH              0.14414
 *O2              0.03969

  * THERMODYNAMIC PROPERTIES FITTED TO 20000.K

    PRODUCTS WHICH WERE CONSIDERED BUT WHOSE MOLE FRACTIONS
    WERE LESS THAN 5.000000E-06 FOR ALL ASSIGNED CONDITIONS

 O3              H2O(cr)         H2O(L)         

 NOTE. WEIGHT FRACTION OF FUEL IN TOTAL FUELS AND OF OXIDANT IN TOTAL OXIDANTS



